challenges-14-plus.yml

```
@@ -0,0 +1,472 @@
+    of users can enter, so I've decided to pop up a big dialog in the front page
+    of that section so that users can *leave* if they don't feel up to it!
+    
+    Here's how it looks!
+    
+    ```html
+    <div id="my_great_modal" style="display: none">
+        <h1>Welcome to our Pizza Section!</h1>
+
+        <p>You need to be a pizza lover before you proceed!</p>
+
+        <button
+            class="btn btn-primary"
+            onclick="javascript:document.getElementById('my_great_modal').style.display='none';"
+        >
+            Proceed
+        </button>
+    </div>
+    ```
+    
+    This works as expected, but I'm a bit nervous about leaving too much time
+    on our users to decide. They should be quick! So I've added a timeout on
+    the main JavaScript of the page to force users *away* after a certain
+    amount of time.
+    
+    ```javascript
+    window.setTimeout(() => {
+        window.location.href = '/';
+    }, 10000);
+    ```
+    
+    How can we use Turbo Drive (instead of just `window.location.href=`)
+    to perform a page visit without doing a full refresh?
+  explanation: |
+    By using the syntax `import * as Turbo from '@hotwired/turbo'` you will be importing all
+    public functions from the library into an object called `Turbo`. This enables
+    you to use some Turbo Drive specific actions in your scripts, like `Turbo.visit`.
+    
+    > You can also use destructuring to import only the functions you need. In
+    > this case, this would be: `import { visit } from '@hotwired/turbo'`.
+    
+    > Turbo is also set automatically as a global variable once you import it anywhere in your
+    > page, so you can also use Turbo from inline scripts!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          ```javascript
+          import Turbo from '@hotwired/turbo`;
+
+          window.setTimeout(() => {
+              Turbo.visit('/');
+          }, 10000);
+          ```
+        b: |
+          ```javascript
+          import * as Turbo from '@hotwired/turbo`;
+
+          window.setTimeout(() => {
+              Turbo.location.href = '/';
+          }, 10000);
+          ```
+        c: |
+          ```javascript
+          import * as Turbo from '@hotwired/turbo`;
+
+          window.setTimeout(() => {
+              Turbo.visit = '/';
+          }, 10000);
+          ```
+        d: |
+          ```javascript
+          import * as Turbo from '@hotwired/turbo`;
+          
+          window.setTimeout(() => {
+              Turbo.visit('/');
+          }, 10000);
+          ```
+      correct: d
+      
+transitions:
+  question: |
+    I just came back from a meeting with our designer, and she has an *amazing* concept
+    for a new transition that we need to add to our page loads between clicks! It looks like
+    this in the CSS main file:
+    
+    ```css
+    body {
+        background-color: #FFF
```

```suggestion
        background-color: #fff
```

I think capital hex letters is considered less-hipster. I double checked tailwind css (currently the max hipster CSS framework) to verify.